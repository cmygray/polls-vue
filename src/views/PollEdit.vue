<template>
  <v-container tag="section" class="poll-edit">
    <h1>Edit Poll</h1>
  </v-container>
</template>

<script lang="ts">
import { Component, Prop, Vue } from 'vue-property-decorator';

@Component
export default class PollEdit extends Vue {
  @Prop()
  id: string;
}
</script>

<style lang="scss"></style>
