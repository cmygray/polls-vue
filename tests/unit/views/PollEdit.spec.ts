import Vuex from 'vuex';
import { mount, createLocalVue } from '@vue/test-utils';
import PollEdit from '@/views/PollEdit.vue';
import '../setup';

const localVue = createLocalVue();
localVue.use(Vuex)

describe('PollEdit.vue', () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = mount(PollEdit, {
      localVue,
      propsData: {
        id: 'blah'
      }
    });
  });

  it('renders as expected', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
