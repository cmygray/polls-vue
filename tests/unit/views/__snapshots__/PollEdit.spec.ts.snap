// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PollEdit.vue renders as expected 1`] = `
<section class="container poll-edit">
  <h1>Edit Poll</h1>
  blah
</section>
`;
